lib/bank_accounts/savings_account.rb

```diff
@@ -0,0 +1,21 @@
+module BankAccounts
+
+  class SavingsAccount < Account
+
+    # ADDS INTEREST TO BALANCE & RETURNS INTEREST AMOUNT
+    def add_interest(rate)
+      interest = @balance * (rate/100)
+      @balance += interest
+      return interest
+    end
+
+    private
+
+    def bank_info
+      @min_balance = 10
+      @atm_withdraw_fee = 2
+    end
+
```

Comment:
Where do you call `bank_info`? And what does it return?

You might consider using Constants instead of instance variables here. Since things like the `min_balance` and `atm_withdraw_fee` are things that don't change (or reflect state) for an instance of a `SavingsAccount`, setting it as a top-level constant might be a better option.